pmm-app/tests/ia/pages/alertRulesPage.js

```diff
@@ -108,9 +120,19 @@ module.exports = {
     closeModal: '$modal-close-button',
     addRule: '$add-alert-rule-modal-add-button',
     cancelAdding: '$add-alert-rule-modal-cancel-button',
-    // editAlertRule returns Edit template button locators for a given source
+    cancelDelete: '$cancel-delete-modal-button',
+    delete: '$confirm-delete-modal-button',
+    // showDetails returns Show rule details button locator for a given rule name
+    showDetails: (ruleName) => `${module.exports.elements.rulesNameCell(ruleName)}//button[@data-qa="show-alert-rule-details"]`,
```

Comment:
Having to refer to `module.exports` seems a bit verbose, we could figure out a shorter syntax.

Reply:
I wasn't able to find a shorter syntax. Could you please point me?

Reply:
Sure, you need to write a helper function `rulesNameCell` and use it everywhere :)

Reply:
done, thanks a lot!